amp/repository/aim/view/scripts/reportWizard/prefilters.js

```diff
@@ -8,384 +8,390 @@ var REPORT_URL = '/rest/data/report/';
 var FILTER_OBJECT = 'workspace-filters-widget-format';
 
-function Filters (filterPanelName, connectionFailureMessage, filterProblemsMessage, loadingDataMessage, 
-				savingDataMessage, cannotSaveFiltersMessage, doReset,settingsPanelName, validationMsgs, embedded) {
-	this.connectionFailureMessage	= connectionFailureMessage;
-	this.filterProblemsMessage		= filterProblemsMessage;
-	this.loadingDataMessage			= loadingDataMessage;
-	this.savingDataMessage			= savingDataMessage;
-	this.cannotSaveFiltersMessage	= cannotSaveFiltersMessage;
-	this.validationMsgs = validationMsgs;
-        this.settingsPanelName=settingsPanelName;
-        this.filterPanelName=filterPanelName;
-    this.additionalParameter="";
-	
-	this.resetString				= "&doreset=true";
-	if ( !doReset )
-		this.resetString	= "";
-	
-	this.filterPanel = new YAHOO.widget.Panel("new", {
-			width:"870px",
-		    fixedcenter: true,
-		    constraintoviewport: true,
-		    underlay:"none",
-		    close:true,
-		    visible:false,
-		    modal:true,
-		    effect:{effect:YAHOO.widget.ContainerEffect.FADE, duration: 0.5},
-		    draggable:true} );
-	this.filterPanel.setHeader(filterPanelName);
-	this.filterPanel.setBody("");
-	this.filterPanel.render(document.body);
-	
-	this.settingsPanel	= new YAHOO.widget.Panel("new2", {
-		width:"450px",
-	    fixedcenter: true,
-	    constraintoviewport: true,
-	    underlay:"none",
-	    close:true,
-	    visible:false,
-	    modal:true,
-	    effect:{effect:YAHOO.widget.ContainerEffect.FADE, duration: 0.5},
-	    draggable:true} );
-	
-	this.settingsPanel.setHeader(filterPanelName);
-	this.settingsPanel.setBody("");
-	this.settingsPanel.render(document.body);
-	
-	this.listFiltersDiv			= document.getElementById("listFiltersDiv");
-	this.hasFilters				= document.getElementById("hasFilters");
-	this.embedded = embedded;
+function Filters(filterPanelName, connectionFailureMessage, filterProblemsMessage, loadingDataMessage,
+                 savingDataMessage, cannotSaveFiltersMessage, doReset, settingsPanelName, validationMsgs, embedded) {
+    this.connectionFailureMessage = connectionFailureMessage;
+    this.filterProblemsMessage = filterProblemsMessage;
+    this.loadingDataMessage = loadingDataMessage;
+    this.savingDataMessage = savingDataMessage;
+    this.cannotSaveFiltersMessage = cannotSaveFiltersMessage;
+    this.validationMsgs = validationMsgs;
+    this.settingsPanelName = settingsPanelName;
+    this.filterPanelName = filterPanelName;
+    this.additionalParameter = "";
+
+    this.resetString = "&doreset=true";
+    if (!doReset)
+        this.resetString = "";
+
+    this.filterPanel = new YAHOO.widget.Panel("new", {
+        width: "870px",
+        fixedcenter: true,
+        constraintoviewport: true,
+        underlay: "none",
+        close: true,
+        visible: false,
+        modal: true,
+        effect: {effect: YAHOO.widget.ContainerEffect.FADE, duration: 0.5},
+        draggable: true

```

Comment:
check if we can get rid of filterpanel

Reply:
Cant remove it because is used also for saving.